{
	"id": "CVE-2019-16056",
	"vulnerabilities": [
		{
			"id": "CVE-2019-16056",
			"references": [
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.alpinelinux.org/vuln/CVE-2019-16056"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "alpine:3.8",
			"vulnerable": true,
			"package": {
				"name": "python3",
				"repositories": [
					"main"
				],
				"architectures": [
					"aarch64",
					"armhf",
					"ppc64le",
					"s390x",
					"x86",
					"x86_64"
				]
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "3.6.8-r1"
					}
				],
				"fixed": [
					"3.6.8-r1"
				]
			}
		}
	],
	"data_source": "alpine-secdb"
}
